### ISSUE TITLE
Support media.githubusercontent.com

### ISSUE THREAD

**Issue by bhaskarvk** [neoascetic/rawgithack#16]
Github serves files stored using git lfs, on media.githubuserconent.com. Please support it on rawgithack

**Comment by neoascetic** [comment]
For what kind of files do you need this? PDF? Something else?

**Comment by bhaskarvk** [comment]
Yeah PDFs, and some big HTMLs. Things that are a few MB in size. Thanks!

**Comment by neoascetic** [comment]
Could you provide example repo, please?

**Comment by bhaskarvk** [comment]
https://media.githubusercontent.com/media/bhaskarvk/UKCrime/master/crimeMap.html from 
https://github.com/bhaskarvk/UKCrime

Warning: That's a 64MB file. That's probably an extreme case example, mostly I think it will be under few megs.

**Comment by neoascetic** [comment]
I am going to do it... one day
